smac/tae/execute_ta_run.py

```diff
@@ -124,9 +129,12 @@ def start(self, config, instance,
                                                           seed=seed,
                                                           instance_specific=instance_specific)
 
-        if self.stats.ta_runs == 0 and status in [StatusType.CRASHED, StatusType.ABORT]:
+        if self.stats.ta_runs == 0 and status == StatusType.CRASHED:
             self.logger.critical("First run crashed -- Abort")
-            sys.exit(1)
+            raise TAEAbortException()
+        if status == StatusType.ABORT:
+            self.logger.error("Target algorithm status ABORT - SMAC will exit.")
```

Comment:
Also, I think this should be the exception message and not a call to the logger.